{{ cookiecutter.__package_name_kebab_case }}/Dockerfile

```
@@ -39,6 +31,27 @@ ENV VIRTUAL_ENV /opt/{{ cookiecutter.__package_name_kebab_case }}-env
 # Set the working directory.
 WORKDIR /workspaces/{{ cookiecutter.__package_name_kebab_case }}/
 
+FROM base as builder
+
+USER root
+
+# Install Poetry in separate venv so it doesn't pollute the main venv.
+ENV POETRY_VERSION 1.6.1
+ENV POETRY_VIRTUAL_ENV /opt/poetry-env
+RUN --mount=type=cache,target=/root/.cache/pip/ \
+    python3 -m venv $POETRY_VIRTUAL_ENV && \
+    $POETRY_VIRTUAL_ENV/bin/pip install -U pip setuptools && \
+    $POETRY_VIRTUAL_ENV/bin/pip install poetry~=$POETRY_VERSION
+RUN ln -s $POETRY_VIRTUAL_ENV/bin/poetry /usr/bin/poetry
```

I suggest to install this to `/usr/local/bin/poetry` instead, which is for user-managed packages (whereas `/usr/bin/` is for system-managed packages):
```suggestion
RUN ln -s $POETRY_VIRTUAL_ENV/bin/poetry /usr/local/bin/poetry
```

---

Fully agree! Fixed